docs/product-business-resource-planner-guide/src/main/asciidoc/AuthoringPlanningAssets-chapter.adoc

```diff
@@ -0,0 +1,8 @@
+[[_planner.authoringPlanningAssets]]
+= Authoring Planning Assets
+:imagesdir: ..
+
+include::AuthoringPlanningAssets/SolverEditor-section.adoc[leveloffset=+1]
+include::AuthoringPlanningAssets/DomainEditor-section.adoc[leveloffset=+1]
+include::AuthoringPlanningAssets/GuidedRuleEditor-section.adoc[leveloffset=+1]
+
```

Comment:
This chapter is not referenced from anywhere, which is confusing. Is it meant to be part of the RHBO guide?

Reply:
No sorry I forgot to delete the old file directory once i was done with it. It's removed now.